ocdata/vasp.py

```diff
@@ -15,7 +15,7 @@
 from ase.calculators.vasp import Vasp2
 from ase.calculators.singlepoint import SinglePointCalculator as SPC
 
-
+# NOTE: this is the setting for slab and adslab
 VASP_FLAGS = {'ibrion': 2,
               'nsw': 200,
```

Comment:
While we're modifying this file, should we change the default to NSW = 2000 as well?

Reply:
Sure, that's a good idea. I will update that